book/integration-service.md

```diff
@@ -0,0 +1,271 @@
+# Integration Service
+
+
+## Overview
+
+The Integration Service is composed of a couple controllers that facilitate automated testing of content produced by the HACBS build pipelines. It is responsible for integration testing capabilities.
```

Comment:
Is it literally a couple of controllers -- i.e. two controllers? I think we can just remove `a couple`.

Reply:
While the number of controllers is currently two, we are in the (ongoing) process of adding additional ones (one for scenario management and one for environment deployments), so a couple might be the most appropriate at this time.

Reply:
I still think it is simpler/clearer to just remove that as it is redundant since `controllers` indicates more than one controller.